src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourcesArr = sourceString.split(';');
+
+  const resultObj = sourcesArr.reduce((acumulator, item) => {
+    const element = item.split(':');
+    const elementName = element[0]?.trim();
+    const elementValue = element[1]?.trim();
+
+    if (elementName && elementValue) {
+      acumulator[elementName] = elementValue;
+    }
 
+    return acumulator;
+  }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. In this case, the code is formatted correctly, but keep this in mind for future reference.